maestro/version.py

```diff
@@ -2,4 +2,4 @@
 # Copyright (C) 2015-2018 SignalFx, Inc.
 
 name = 'maestro-ng'
-version = '0.7.5'
+version = '0.8.0'
```

Comment:
Remove this, the release takes care of that (I'll do the release).

Reply:
ack